### ISSUE TITLE
Netflix 4K Support

### ISSUE THREAD

**Issue by pclavelloux** [ThatGuySam/doesitarm#42]
For a great example issue check out https://github.com/ThatGuySam/doesitarm/issues/31

**The official name of the app**
Netflix

**Comment by ThatGuySam** [comment]
Thanks for the App Request! 

I'm currently working on getting to all the requests, however, if you'd like help push things along feel free to add context based on the [App Request Template](https://github.com/ThatGuySam/doesitarm/blob/master/.github/ISSUE_TEMPLATE/app-request-template.md)

**Comment by Drakefly** [comment]
Hello, I think it's not an application to check because netflix is fully usable from a browser or from the appStore. So netflix will works.

**Comment by Zlitus** [comment]
Is 4K working on the web version of Netflix with Safari? It is working with some MacBook Pro T2, I would be interested to know if it also work with M1.

**Comment by ThatGuySam** [comment]
Sometimes you have to configure the settings because Netflix has been fiddling around with the default to reduce the bandwidth they have to serve. 

You can learn more here: https://www.digitaltrends.com/home-theater/getting-hd-netflix/

There shouldn’t be really any difference between Intel and ARM processors when using the Netflix website.

**Comment by Zlitus** [comment]
Well actually no: Netflix in 4K is only compatible with a few browsers/os/devices:
- Microsoft Edge on Windows 10, with a supported GPU
- Safari on macOS, but only on Big Sur and on a Mac with a T2 chip

Here an article about it:
- https://www.macrumors.com/2020/10/01/macs-need-t2-security-chip-4k-netflix-big-sur/

So, to me, it would make sens to test it with the new CPU/GPU M1 cause it is not necessarily still compatible with it.

**Comment by ThatGuySam** [comment]
That's interesting. 

We'll have to get someone with an Apple SIlicon chip to test it out and see what the differences are

**Comment by Zlitus** [comment]
Ok, I just received my MacBook Pro M1 this morning and been on Netflix and test the CTRL+Shift+Option+D “debug panel”, and Netflix never send me better than 1080p stream. I've a fiber internet connection, with 255Mbps according to the Netflix connection speed test.

The native resolution is 2880x1800, it would make sense to receive a bigger stream. Changing the macOS Scale or the zoom level in Safari don't change anything, I don't have a secondary 4K screen to test but it doesn't look good.

Screenshot here: https://imgur.com/a/rLwNlxQ
Let's notice the UA, still indicating “Intel”, probably for compatibility reasons. I don't know how to "force 4K" but Netflix don't seem ready to send it anyway.

**Comment by ThatGuySam** [comment]
It looks like there are several outlets confirming this

https://9to5mac.com/2020/06/29/safari-can-now-stream-4k-hdr-and-dolby-vision-content-on-netflix-with-macos-big-sur/

https://www.theverge.com/2020/6/29/21307620/apple-mac-netflix-4k-hdr-macos-big-sur-dolby-vision-safari

If you'd like we can wait until some with a 4k monitor confirms or I can list it now

**Comment by ThatGuySam** [comment]
Here's the info we'll use

The official name of the app
Netflix 4K

Proposed App Status
Yes, Netflix now supports 4k on macOS Big Sur (but you don't need Apple Silicon)

Related Issue Tracker Link or discussion
https://9to5mac.com/2020/06/29/safari-can-now-stream-4k-hdr-and-dolby-vision-content-on-netflix-with-macos-big-sur/

https://github.com/ThatGuySam/doesitarm/issues/42


An Official App Download Link
https://www.netflix.com/

**Comment by Zlitus** [comment]
Well: At the moment, as far as I know, it working only on Macs with a T2 chip, which concern only theses devices:
Mac Pro 2019
Mac Mini 2018
MacBook Air 2018+
MacBook Pro 2018+

And, we need a confirmation from someone with a Macbook M1 & a 4K screen, but it seem like the new generation can't (yet) use 4K on Netflix.

**Comment by moffpage** [comment]
Close this issue please

**Comment by ThatGuySam** [comment]
@pclavelloux @Zlitus Any objections?

**Comment by Zlitus** [comment]
@pclavelloux  Yes unfortunately I have an objection. Netflix does not support 4K on M1 Mac (yet), the status didn't changed since my last messages and the links sent since are older and talk about Intel Macs.

Netflix support 4K on Mac for Big Sur and Safari only, and only with some specific hardware witch not include (yet) the M1 CPU/GPU as far as I know. If somebody as more information or can proof than Netflix run on 4K on M1, feel free to share it.

**Comment by ThatGuySam** [comment]
So perhaps we list this as: 
Netflix 4K Support on Safari

**Comment by Zlitus** [comment]
Well, the problem is that no other browsers or apps for macOS (M1 or Intel) offer 4K for Netflix. So for now it mean "Netflix 4K on M1 is not available"

**Comment by moffpage** [comment]
Close this issue please

**Comment by Zlitus** [comment]
> Close this issue please

Can you please elaborate and explain why we should close this issue? For what I see, we don't have (yet) any way to see Netflix in 4K on Apple M1, and we don't have any reason to think than Netflix is not working on fixing that. So the issue should stay open to follow the potential progress.

**Comment by ThatGuySam** [comment]
There's no harm in leaving it open for now.

**Comment by ThatGuySam** [comment]
I've found one way to check the current resolution Netflix is playing at within your browser by getting the resolution of the video HTML element. 

Run the following in the respective Dev Tools console for your browser:
> document.querySelector('video').videoHeight

This should be the maximum possible resolution the video content is currently playing at.

**Comment by moffpage** [comment]
Gonna try that tomorrow

**Comment by ThatGuySam** [comment]
Thanks @pclavelloux for opening this and @Zlitus for clarifying the difference between ordinary playback and 4K! Netflix documents 4K support on qualifying Apple Silicon Macs.

Requirements for 4K streaming, from [Netflix's official Mac guidance](https://help.netflix.com/en/node/55764):

- A qualifying Apple Silicon Mac with macOS 11 or later and the latest Safari.
- A Netflix plan that supports Ultra HD and a compatible display.
- A stable connection of at least 15 Mbps.
- Playback quality set to Auto or High.
- External displays must support 4K at 60 Hz and HDCP 2.2.

These conditions describe Safari playback. They do not verify a standalone Netflix app or guarantee 4K on every display setup.

Listing: [Netflix 4K on Safari on Does It ARM](https://doesitarm.com/app/netflix-4k-on-safari).

The [wrapper-data cleanup](https://github.com/ThatGuySam/doesitarm/commit/eab6b2a4c24a5797d1d77a2edd2e32285f53f83b) was verified live on September 7, 2026. This verifies the listing, not a hands-on playback test.

The [hero copy update](https://github.com/ThatGuySam/doesitarm/commit/b53be677fa142afbd948ac5e654d576b37bc931f) shortens the headline to "✅ Yes, 4K streaming supported" and links directly to these requirements. The shorter headline and requirements link are verified live. Closing this listing request after maintainer review.

Feel free to [browse the app list](https://doesitarm.com) and [suggest another app](https://github.com/ThatGuySam/doesitarm/issues/new?template=app-request-template.yml) you'd like us to review!
